core/notify.py

```diff
@@ -1,20 +1,13 @@
 import time
-
-from core.rovecomm import RoveCommEthernetUdp, RoveCommPacket
-
-
-DRIVE_BOARD_IP = "192.168.1.142"
-NOTIFY_ID = 7000
-
+import core
 
 # TODO: Wrap this in a Lighting/Media Board interface
 class Notify:
-
     def __init__(self, rove_comm):
         self.rove_comm_node = rove_comm
 
     def _notify(self, val):
-        self.rove_comm_node.write(RoveCommPacket(NOTIFY_ID, 'b', (val, 0), ip_octet_4='142'))
+        self.rove_comm_node.write(core.RoveCommPacket(core.NOTIFY_ID, "b", (val, 0), ip_octet_4="142"))
 
     def notify_finish(self):
```

Comment:
This code hasn't been touched for a while (since Valkyrie iirc) and was just updated due to global edits. I need to come back and rewrite this at a later point hence my TODO. I'll touch this in a later PR>